packages/kie-bc-editors/src/jslienzo/JsLienzo.ts

```diff
@@ -14,21 +14,68 @@
  * limitations under the License.
  */
 
+/**
+ * Javascript API that allows interactions with nodes and connectors
+ * in Kogito editors.
+ */
 export interface JsLienzo {
+  /**
+   * Returns ID attributes of all nodes displayed in editors canvas.
+   */
   getNodeIds(): string[];
 
+  /**
+   * Returns a background color of a node with provided UUID.
+   * Returned string is a hex number of the color.
+   *
+   * @param UUID ID attribute of the queried node
+   */
   getBackgroundColor(UUID: string): string;
 
+  /**
+   * Sets a background color of a node with provided UUID.
+   *
+   * @param UUID ID attribute of the target node
+   * @param backgroundColor hex number of the desired color as string
+   */
   setBackgroundColor(UUID: string, backgroundColor: string): void;
 
+  /**
+   * Returns a border color of a node with provided UUID.
+   * Returned string is a hex number of the color.
+   *
+   * @param UUID ID attribute of the target node
+   */
   getBorderColor(UUID: string): string;
 
+  /**
+   * Sets a border color of a node with provided UUID.
+   *
+   * @param UUID ID attribute of the target node
+   * @param backgroundColor hex number of the desired color as string
+   */
   setBorderColor(UUID: string, backgroundColor: string): void;
 
+  /**
+   * Returns a canvas location of a node with provided UUID.
+   * Returns an array where first position is X-attribute
+   * and second position is Y-attribute.
+   *
+   * @param UUID ID attribute of the target node
+   */
   getLocation(UUID: string): number[];
 
+  /**
+   *
+   * @param UUID
+   */
   getAbsoluteLocation(UUID: string): number[];
```

Comment:
What is the difference between Aboslute and regular one?

Reply:
The getLocation method is supposed to return node coordinates within the cavas and the getAbsoluteLocation method is supposed to return  node coordinates in the window. 
I see they both methods are returning same values, it is definetly an issue we have to check. However it does not affect this PR itself the issue is in the Lienzo side. 

Maybe it is a good idea to mention it in the PR: 
https://github.com/kiegroup/kogito-editors-java/pull/117
Cc: @inodeman

Reply:
Thank you @handreyrc much appreciated. Will do!